client/src/app.js

```diff
@@ -35,6 +36,7 @@ class App extends Component {
           <SignupPage path="signup" />
           <InactivePage path="inactive" />
           <NotFoundPage default />
+          <PrivateRoute as={UserProfilePage} path="userprofile/:authorId" />
```

Comment:
https://github.com/ExiaSR/hyperion/pull/135#discussion_r268499827

Check out my previous comment for URL convention